themes.json

```diff
@@ -1,5 +1,17 @@
 [
 {
+    "Title": "Nature",
+    "FileName": "Nature.tmTheme",
+    "Author": "Robert Englund",
+    "Description": "A theme that has nice colors like nature in it!"
+},
+{
+    "Title": "Ubunty Style",
+    "FileName": "UbuntyStyle.tmTheme",
+    "Author": "Robert Englund",
+    "Description": "A theme that has Ubuntys color scheme!"
```

Comment:
Should I change it?
And can I just upload the changed to my fork directly it changes here aswell?